### PR TITLE
cdbfast regexp test migration to ICG

### PR DESCRIPTION
**Pull request by oarap**
@vraghavan78 @xinzweb @hsyuan Please take a look at this test for review.

query01, query02, query06, query08, query09, query10, query12 were part of the ICG and removed from this suite.

### CODE REVIEW

**Comment by hsyuan** [comment]
LGTM

File: src/test/regress/greenplum_schedule [hunk 1]
```diff
@@ -87,9 +87,9 @@ test: bfv_cte bfv_joins bfv_statistic bfv_subquery bfv_planner bfv_legacy
 
 test: qp_executor qp_olap_windowerr qp_olap_window qp_derived_table qp_bitmapscan
 
-test: qp_correlated_query
+test: qp_correlated_query
```
Comment (danielgustafsson) [inline src/test/regress/greenplum_schedule]: Stray whitespace

**Comment by danielgustafsson** [comment]
Nothing sticks out, looks good to me.

**Comment by oarap** [comment]
Merged aec36ca1d364f774af7bcaba79b374f8405dae5a

**Pull request closed without merging**